back_end/src/main/java/com/back_end/back_end/Services/StudentServiceImpl.java

```diff
@@ -26,37 +30,53 @@
     List<Student> studList = studentRepository.findAll();
     return studList;
 
+  
+}
+  @Override
+  public Student create(Student student) {
+
+    Sequence sequence = sequenceServices.getNextSequence(Constants.SEQUENCE_TYPE_ROLL_NO);
+    if (null != sequence) {
+      String rollNo = sequence.getRollNo();
+      Sequence newSequence = new Sequence(sequence.getId(),sequence.getType(), sequence.getPrefix(),sequence.getNext_sequence() + 1);
+      sequenceServices.saveSequence(newSequence);
+      return studentRepository.save(new Student(rollNo, student.getName(), student.getDob(), student.getStud_class(),
+      student.getDivision(), student.getGender()));
+    } else {
+      Sequence newSequence = new Sequence();
+      newSequence.setType(Constants.SEQUENCE_TYPE_ROLL_NO);
+      newSequence.setPrefix(Constants.SEQUENCE_PREFIX_ROLL_NO);
+      newSequence.setNext_sequence(101);
+      sequenceServices.saveSequence(newSequence);
+      String rollNo = Constants.SEQUENCE_PREFIX_ROLL_NO + "-" + 100;
+      return studentRepository.save(new Student(rollNo, student.getName(), student.getDob(), student.getStud_class(),
+          student.getDivision(), student.getGender()));
+
+    }
+    
+
+
   }
 
   @Override
-  public String create(Student student) {
+  public String update(String id, Student student) {
 
-    studentRepository.save(new Student(student.getRollno(), student.getName(), student.getDob(),
-        student.getStud_class(), student.getDivision(), student.getGender()));
+    Optional<Student> studentData = studentRepository.findById(id);
 
-    return "successfull";
+    if (studentData.isPresent()) {
+      Student _Student = studentData.get();
+      _Student.setRollno(student.getRollno());
 
-  }
+      _Student.setName(student.getName());
+      _Student.setDob(student.getDob());
+      _Student.setStud_class(student.getStud_class());
+      _Student.setDivision(student.getDivision());
+      studentRepository.save(_Student);
+
+      return "successfull";
```

Comment:
Return  the response from        studentRepository.save(_Student); instead of these string. Refer the create function for correct method.